website/content/specification/syntax/instances.md

```diff
@@ -0,0 +1,466 @@
+
+##### `@in-xml`
+
+The optional `@in-xml` attribute controls the representational form of a group of instances in XML.
+
+When no attribute and value is provided for the `@in-xml` attribute, the value MUST default to `UNGROUPED`.
+
+One of the following behaviors MUST be used based on the provided, or default value when no attribute and value is provided.
+
+| Value | XML Behavior |
+|:--- |:--- |
+| [`GROUPED`](#in-xmlgrouped) | The child elements MUST be placed within a wrapper element with a local name equal to the value of the `@name` attribute. Each child element's local name will be the [effective name](#naming-and-use-name) of the instance. |
+| [`UNGROUPED`](#in-xmlungrouped) | The child elements MUST appear without a grouping (wrapper) element. The `group-as/@name` is ignored. Each child element's local name will be the [effective name](#naming-and-use-name) of the instance. |
+
+###### `@in-xml="GROUPED"`
+
+TODO: P2: complete this section.
+
+###### `@in-xml="UNGROUPED"`
+
+TODO: P2: complete this section.
+
+##### `@name`
+
+The REQUIRED `@name` attribute declares the grouping name to use in JSON, YAML and XML (when exposed).
+
+In JSON and YAML, this name is used as the property name.
+
+In XML, the specific use of the `@name` is based on the `@in-xml` attribute's value.
+
+{{<callout>}}
+While not required, use of a plural form of the [effective name](#naming-and-use-name) for the instance is a general convention applied to `<group-as>` naming.
+{{</callout>}}
+
+### Instance Naming
+
+Given the set of instance names for a *definition*, a given *instance* on that definition MUST have a unique effective name. If the instance allows a [maximum cardinality](#max-occurs) greater than `1`, the [`<group-as name>`](#name) MUST also be unique within the set of *instance names* for the *definition*.
+
+For *field instance* values, the effective name of the value, based on a `<json-value-key>` or `<json-value-key-flag>` MUST also be unique within the set of *instance names* for the *definition*. This ensures that the property name for the *field* value is unique in JSON and YAML.
+
+For *field instances* that use [`@in-xml="UNWRAPPED"`](#in-xml-1), no other effective instance name in the within the set of *instance names* for the *definition* MUST be named in a way that matches the allowed element contents of the [markup-multiline data type](/specification/datatypes/#markup-multiline) in XML. This ensures that a naming clash with markup data is disallowed.
```

Comment:
Another Issue to go with #366: align Metaschema Schematron checks with these rules.